src/panels/config/integrations/integration-panels/ozw/dialog-ozw-refresh-node.ts

```diff
@@ -0,0 +1,292 @@
+import {
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  internalProperty,
+  TemplateResult,
+  PropertyValues,
+  css,
+} from "lit-element";
+import "../../../../../components/ha-code-editor";
+import "../../../../../components/ha-circular-progress";
+import { createCloseHeading } from "../../../../../components/ha-dialog";
+import { haStyleDialog } from "../../../../../resources/styles";
+import { HomeAssistant } from "../../../../../types";
+import { OZWRefreshNodeDialogParams } from "./show-dialog-ozw-refresh-node";
+
+import {
+  OZWNodeIdentifiers,
+  getIdentifiersFromDevice,
+  fetchOZWNodeMetadata,
+  OZWDeviceMetaData,
+  OZWDevice,
+} from "../../../../../data/ozw";
+import { DeviceRegistryEntry } from "../../../../../data/device_registry";
+
+@customElement("dialog-ozw-refresh-node")
+class DialogOZWRefreshNode extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property()
+  private node_id = 0;
+
+  @property()
+  private ozw_instance = 1;
+
+  @internalProperty() private _device?: DeviceRegistryEntry;
+
+  @internalProperty() private _nodeMetaData?: OZWDeviceMetaData;
+
+  @internalProperty() private _node?: OZWDevice;
+
+  @internalProperty() private _active = false;
+
+  @internalProperty() private _complete = false;
+
+  @internalProperty() private _progress = [
+    "ProtocolInfo",
+    "Probe",
+    "WakeUp",
+    "ManufacturerSpecific1",
+    "NodeInfo",
+    "NodePlusInfo",
+    "ManufacturerSpecific2",
+    "Versions",
+    "Instances",
+    "Static",
+    "CacheLoad",
+    "Associations",
+    "Neighbors",
+    "Session",
+    "Dynamic",
+    "Configuration",
+    "Complete",
+  ];
```

Comment:
This doesn't have to be a `internalProperty` or even be in the class, extract it out of the class as a const, or even better to `data/ozw`.